hieradata_aws/common.yaml

```diff
@@ -986,6 +986,7 @@ icinga::client::config::allowed_hosts: "10.0.0.0/8"
 icinga::config::http_username: "%{hiera('http_username')}"
 icinga::config::http_password: "%{hiera('http_password')}"
 
+icinga::config::graphite_host: 'graphite'
```

Comment:
I think we've been using `_hostname` instead of `_host` to reference the AWS URL overrides